lib/netbox_client_ruby/api/dcim/rack_role.rb

```diff
@@ -0,0 +1,16 @@
+require 'netbox_client_ruby/entity'
+require 'netbox_client_ruby/api/dcim/region'
+require 'netbox_client_ruby/api/tenancy/tenant'
+
+module NetboxClientRuby
+  module DCIM
+    class RackRole
+      include Entity
+
+      id id: :id
```

Comment:
This syntax look a bit weird to me 🙂

Reply:
At least it's consistent with other files.